docs/workflow/requirements/linux-requirements.md

```diff
@@ -16,12 +16,42 @@ Install Docker. For further installation instructions, see [here](https://docs.d
 
 ## Environment
 
-These instructions are written assuming the current Ubuntu LTS. Pull Requests are welcome to address other environments.
+Below are the requirements for toolchain setup, depending on your environment. Pull Requests are welcome to address other environments.
 
 Minimum RAM required to build is 1GB. The build is known to fail on 512 MB VMs ([dotnet/runtime#4069](https://github.com/dotnet/runtime/issues/4069)).
 
 ### Toolchain Setup
 
+#### Fedora
```

Comment:
Ubuntu is the most popular distro. It should be first in the doc. The instructions for other distros should be at the end.